h2o-core/src/main/java/water/parser/ExportParquet.java

```diff
@@ -0,0 +1,9 @@
+package water.parser;
+
+import water.H2O;
+import water.fvec.Frame;
+
+public interface ExportParquet{
```

Comment:
this interface could be generic, eg. BinaryFormatExporter

Reply:
this was fixed here: [bca2bf8](https://github.com/h2oai/h2o-3/pull/6190/commits/bca2bf83f7b5828b4a9950d36c6cc5827e0fe92b)